manifests/mod/passenger.pp

```diff
@@ -963,22 +649,7 @@
   # - $passenger_use_global_queue : since 2.0.4.Deprecated in 4.0.0.
```

Comment:
If it's deprecated, should it be removed?

Reply:
done! Thanks @ekohl for your quick responses and review.